tests/Examples/ArgsAnotherArgsTest.php

```diff
@@ -9,7 +9,7 @@ class ArgsAnotherArgsTest extends TaskTestCase
     // args:another-args
     public function test(): void
     {
-        $process = $this->runTask(['args:another-args', 'FIXME(required)', '--test2', 1]);
+        $process = $this->runTask(['args:another-args', 'FIXME(required)', '--test2', 1, '--no-trust']);
```

Comment:
Is there a way to not update all tests?

Reply:
Did not find a way, so I'm open to any suggestions